lib/trento.pm

```diff
@@ -768,11 +768,12 @@ Get the api-key from the Trento installation
 
 sub trento_api_key {
     my ($wd, $basedir) = @_;
-    my $cmd = join(' ', $basedir . '/trento-server-api-key.sh',
+    my $cmd = join(' ', $basedir . 'trento_deploy/trento_deploy.py',
```

Comment:
I see in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/trento/install_agents.pm#L19-L23 that this is called with `$basedir = '/root/test'`, so unless we expect a directory called `/root/testtrento_deploy/...` to exist, then the line above is missing a `/` right after `$basedir`.

I was going to check how that's working in the VRs, but they have not been added yet.

Reply:
Good obervation master thx, refactored